addon/mixins/axis-titles.js

```diff
@@ -0,0 +1,174 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
```

Comment:
I agree with Phil, I think that dramatically improves the ease of understanding for what the files does.